docs/installation.rst

```diff
@@ -0,0 +1,41 @@
+Installation
```

Comment:
This now has some redundancies with the `Readme.md`. If you want to follow-up, you can have a look at the structure of https://github.com/MannLabs/alphadia?tab=readme-ov-file#installation where we had the same problem.

Reply:
Will address this in another pull request :+1: